aws/cluster/variables.tf

```diff
@@ -683,6 +683,18 @@ variable "kubernetes-cpu-cfs-quota-period" {
   default     = "100ms"
 }
 
+variable "serialize-image-pulls-enabled" {
+  type        = "string"
+  description = "Boolean (true or false) enable or disable serializeImagePulls (serialize-image-pulls). If disabled Docker default download concurrency is 3."
+  default     = "true"
```

Comment:
So those default values are current defaults as used by kubelet?

Reply:
Correct, these are the defaults.